lib/RpmSpecMaker.pm6

```diff
@@ -105,20 +105,26 @@ module RpmSpecMaker {
     }
 
     sub requires(:$meta!) is export {
+        return "Requires:       perl6 >= 2016.12" if not $meta<depends>;
```

Comment:
I'd rather you change it to a `if $meta<depends>` like `build-requires`, so `requires` and `build-requires` have a similar structure and to avoid the duplication of the "how a dependency is stated in a spec file" information and the minimum perl6 version.

Reply:
Thanks. In general I try to return early and often. I think that is better for readability and performance.
Anyways, in this case you have right. Looks better this way. Changed.